src/calculus/HCAT.jl

```diff
@@ -37,23 +37,22 @@ julia> H*(ones(3),ones(3))
 
 """
 struct HCAT{M, # number of codomains  
-	    N, # number of AbstractOperator 
-	    L <: NTuple{N,AbstractOperator},
-	    P <: NTuple{N,Union{Int,Tuple}},
-	    C <: Union{NTuple{M,AbstractArray}, AbstractArray},
-	    } <: AbstractOperator
+            N, # number of AbstractOperator 
+            L <: NTuple{N,AbstractOperator},
+            P <: NTuple{N,Union{Int,Tuple}},
+            C <: Union{NTuple{M,AbstractArray}, AbstractArray},
+           } <: AbstractOperator
 	A::L     # tuple of AbstractOperators
 	idxs::P  # indices 
```

Comment:
Also, I don’t remember what the advantage of storing these indices is, but I guess that storing the exact Tuple of operators to be horizontally concatenated could potentially simplify the code?

Reply:
The indices are due to eventual rearrangement of block of variables (I guess we should call them ArrayPartitions now). I suppose stuff can be simplified there as well, but it's out of the scope of this PR.